bigbluebutton-html5/imports/api/2.0/users/server/modifiers/userEjected.js

```diff
@@ -0,0 +1,39 @@
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+import Users from '/imports/api/2.0/users';
+
+export default function userEjected(meetingId, userId) {
```

Comment:
This modifier does exactly the same thing as the `removeUser`.
Maybe just change the `ejected` and reuse the `removeUser` method.